marmoset/__init__.py

```diff
@@ -1,6 +1,9 @@
-from . import config, cli
+from marmoset import config
+from marmoset import cli
+from marmoset import validation
 
-def run(config_file = None):
+
+def run(config_file=None):
     cfg = config.load(config_file)
```

Comment:
it is only if you use a other configfile than default
later in the code the config file path will automated be set by `path.join(path.dirname(__file__), '../marmoset.conf')` if no configfile parameter was provided